using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -126,10 +118,111 @@ Set the Push Identifier request
 }
 ```
 
+### Analytics Identity Response
+
+#### Event Description
+
+This event is dispatched by the Analytics extension upon receiving the Analytics identity request event that gets generated in the Analytics [getTrackingIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#gettrackingidentifier) and [getVisitorIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#getvisitoridentifier-1) calls. This event contains the unique tracking identifier. This value may be null if identity extension is enabled and there is a valid ECID generated.
```

Comment:
Instead of adding the description of the Analytics Identity response event here, I recommend adding a short description about why is the Identity extension listening for this event and adding a link to the https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-event-reference#analytics-response-identity